stack_exchange_api.py

```diff
@@ -14,7 +15,8 @@ def get_authorization_url():
     oauth = OAuth2Session(client_id, redirect_uri=redirect_uri, scope=scope)
     authorization_url, state = oauth.authorization_url('https://stackexchange.com/oauth/dialog')
 
-    print("Access the following URL to obtain the access token:", authorization_url)
+    logging.basicConfig(level=logging.INFO)
+    logging.info("Access the following URL to obtain the access token: ", authorization_url)
```

Comment:
Here you are logging a variable data (Chapter `Logging Variable Data` from the [guide](https://realpython.com/python-logging/).
So any of these two approaches should be fine:

_string concatenation with the authorization_url (notice the + sign, instead of ,)_:
```
logging.info("Access the following URL to obtain the access token: " + authorization_url)
```
_or, to be more fancy, using string formatter_:
```
logging.info("Access the following URL to obtain the access token: %s", authorization_url)
```